composer.json

```diff
@@ -30,7 +30,7 @@
         "doctrine/doctrine-fixtures-bundle": "~2.3",
         "phpunit/phpunit": "^4.8.35|^5.7|^6.1",
         "twig/twig": "~1.12|~2.0",
-        "nelmio/alice": "~1.7|~2.0",
+        "nelmio/alice": "~1.7|~2.0|~3.0",
```

Comment:
I would go strait to `~3.1` as there is a good fix in it

Reply:
Are we sure that we want to support 3 major versions at the same time? Isn't this creating us a lot of issues with cross-compatibility? Also, IMHO we should re-target this PR to the 2.x branch, so we can use some breaking changes to solve this easily.

Reply:
@Jean85 I rebase the related pr for that: https://github.com/liip/LiipFunctionalTestBundle/pull/337